### DISCUSSION TITLE
while updating cockpit got this (how to fix)

### DISCUSSION THREAD

**Discussion by rzmuda** [cockpit-project/cockpit#19650] [Q&A] [answered]
Loading available updates failed
E: The repository 'cdrom://Ubuntu 22.04.3 LTS _Jammy Jellyfish_ - Release amd64 (20230807.2) jammy Release' does not have a Release file. W: Updating from such a repository can't be done securely, and is therefore disabled by default. W: See apt-secure(8) manpage for repository creation and user configuration details. E: cdrom://Ubuntu 22.04.3 LTS _Jammy Jellyfish_ - Release amd64 (20230807.2) jammy Release is not (yet) available (Please use apt-cdrom to make this CD-ROM recognized by APT. apt-get update cannot be used to add new CD-ROMs) E: The repository 'https://ppa.launchpadcontent.net/mc3man/xerus-media/ubuntu jammy Release' does not have a Release file. W: Updating from such a repository can't be done securely, and is therefore disabled by default. W: See apt-secure(8) manpage for repository creation and user configuration details.

Please reload the page after resolving the issue.

**Comment by martinpitt** [comment, marked as answer]
None of this is related to cockpit. You have some obsolete/invalid PPAs, please remove them.
